packages/react/src/components/Spinner/Spinner.tsx

```diff
@@ -41,7 +41,7 @@ const variantMap: Record<Variant, { foreground: string; background: string }> =
 
 export type SpinnerProps = {
   /** Spinner title  */
-  title: string;
+  title?: string;
```

Comment:
I don't see a problem with this, but I would suggest its done a separate PR for traceability. We automate our changelogs where each PR is a separate entry so its also clearer for users of changes made.